db/migrate/20170807145023_create_timelogs.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTimelogs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :timelogs do |t|
+      t.datetime :start_time
+      t.integer :duration, null: false, default: ""
+      t.integer :user_id
```

Comment:
This is a reference. If we think logically - `Timelog` can't exist without relation to user. It means that we need to add `foreign_key constraint` here. Also, it is a good practice to add indexes for foreign keys.